### ISSUE TITLE
[Forge - 1.19.2] JEI dependency version mismatch causes crash upon launch

### ISSUE THREAD

**Issue by julkoh** [emilyploszaj/emi#780]
Using latest releases of EMI and JEI for version 1.19.2 : 

- Forge : 43.4.0
- EMI : 1.1.17
- JEI : 11.6.0.1019

I get the following error in logs : 

```
[13:49:46] [main/ERROR]: Unsupported installed optional dependencies:
	Mod ID: 'jei', Requested by: 'emi', Expected range: '[15.20.0.0,)', Actual version: '11.6.0.1019'
```

Changing the minimum version in `mods.toml` fixed the issue.

**Comment by Yarden-zamir** [comment]
Same here, also 1.19.2

**Comment by theboo** [comment]
looks like 1.19.2 emi accidentally got the 1.20.1 versions of jei set in the mods.toml in the latest release

**Comment by emilyploszaj** [comment]
Oops. Resolved in 1.1.18
